src/fastga/models/aerodynamics/external/openvsp/openvsp.py

```diff
@@ -1443,6 +1498,10 @@ def compute_wing_rotor(self, inputs, outputs, altitude, mach, aoa_angle, thrust,
             parser.transfer_var(float(l4_wing), 0, 5)
             parser.mark_anchor("sweep_0_wing")
             parser.transfer_var(float(sweep_0_wing), 0, 5)
+            parser.mark_anchor("twist")
+            parser.transfer_var(float(twist), 0, 5)
+            parser.mark_anchor("dihedral_angle")
+            parser.transfer_var(float(dihedral_angle), 0, 5)
```

Comment:
I agree but we need to find time to do it properly, as we could save a lot of lines and computation time using for instance @functools.lru_cache and refactoring of functions, but that would require a rework from the ground up which would take quite some times. Feel free to open an issue so that we don't forget about it.